guessingGame.js

```diff
@@ -2,37 +2,69 @@ let newGuessingGame = require('./lib/newGuessingGame');
 let guessingGameMakeGuess = require('./lib/guessingGameMakeGuess');
 let guessingGameIsDone = require('./lib/guessingGameIsDone');
 
-let wordToGuess = 'hello';
+let process = require('process');
 
-let game = newGuessingGame('hello');
+let readlineSync;
+try {
+  readlineSync = require('readline-sync');
+} catch (err) {
+  console.log('You must install the readline-sync module. Run the following');
+  console.log('command to install it:');
+  console.log();
+  console.log('  npm install');
+  console.log();
+
+  process.exit();
+}
+let secretInput = readlineSync.question('Enter a secret word: ', { hideEchoBack: true });
+
+//console.log(`You entered: ${secretInput}`);
```

Comment:
Unless they're here to help me understand how you're thinking about the code, remove the random, comment-out `console.log` statements before pushing.